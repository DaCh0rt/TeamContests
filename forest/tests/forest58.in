233 592
5125 3570
-3401 -1043
-3462 -6750
-3119 5470
-8169 2423
7807 -6720
-2019 -5039
-23 8372
-2043 -183
8771 6411
7649 4991
8832 -3822
4000 -420
1993 -1867
-6352 9356
-2719 -619
-4115 -7673
-6259 5943
8834 -3043
3831 -3376
-3563 -1986
8533 4631
6533 -3447
5122 8129
-214 8983
2807 3826
-8375 -5399
-9988 7066
6466 2505
5405 -2881
7675 8565
-4032 1217
-4619 -1503
1714 -187
-7886 7809
7023 -2867
1638 578
-6049 5996
9270 -66
9444 -480
-9849 -191
-4270 -6611
9411 -7431
-9800 -164
-4327 -3873
3034 7298
934 4735
9816 -919
8811 -1911
1425 4303
3766 3995
-9905 -8562
-2690 -8147
5478 -9888
7395 -1306
8271 -978
1675 8186
8483 -6300
8141 -5264
9692 2382
-6918 908
-7987 -8629
6678 6428
6550 -1711
-7804 -5620
-2078 -1693
8990 -7701
-50 4592
-3271 -9243
1617 4607
9725 2515
-2786 -2915
7220 678
1611 9149
9267 -8359
3862 -6290
6135 -8931
-7262 7331
6153 -9153
75 7348
-9958 1946
-9575 7026
-9454 -6246
4583 7888
-8314 -6191
-8299 5212
-9000 -2763
-861 -1571
-8766 9630
0 1554
-6794 9941
2197 -551
6004 -1463
6227 -1656
-374 -5551
4399 3234
-8448 -909
-8686 -9018
4038 -5480
-471 -1190
5699 622
-9281 -5609
-6976 7096
5062 -3527
-7042 6048
8612 3753
-8181 6161
-1929 -6319
4863 -2085
6441 -3470
4066 -1308
-2137 4298
8005 907
3773 6083
-329 -7713
5992 -596
8955 -7642
-6791 2847
3931 -3575
-918 8065
3681 7112
8939 -1503
-1200 -2698
-7276 -5830
-1677 -3460
30 -2705
-7525 -1977
6664 1435
2787 -35
5524 -3859
8315 -5664
8257 3193
-5674 5730
8096 -3055
1916 9591
-3935 4468
1010 2642
-1803 -6232
6887 -462
-2546 -5739
2412 -3486
-1678 2618
-6173 -8977
3864 2470
6523 8057
-2 -8527
-6305 2217
-8341 2828
4954 4275
7508 3050
-7842 8751
-5305 -3218
-9634 -6342
-2577 9898
8756 -675
-2288 -5169
-4899 3611
-3223 9895
2507 -6194
4955 5670
9044 -726
9616 5830
-1424 2380
-1828 8032
-2822 -3010
1589 -2524
-4289 6866
-2489 -9187
6157 2000
-4272 7780
6306 3091
115 -9687
-3776 8497
-5908 -4949
-1449 -6594
-9492 4055
1637 526
857 -2779
1362 -8563
339 -8745
9614 1969
-5172 9500
-3773 -1661
-2522 -2797
1316 -9616
-8861 -1847
1227 -3154
-1992 9917
-7268 8865
9959 -8163
-1712 1292
-6890 -6877
-2709 -4072
-6524 1451
1049 936
-3633 -507
-6352 -2784
-7531 1288
8714 -6965
-7605 2023
-1361 -9604
-6920 -1604
-3685 -4021
9820 -5331
2048 -9889
9826 -4802
-5225 4194
-8165 3431
8332 8807
3885 6725
4099 -7986
-6513 -8448
-4761 -6381
-9216 4103
-990 1979
5926 -6565
-7002 -1720
-4313 3250
5487 8494
9136 -3890
-2499 -5069
9256 1793
6377 7574
-6859 -8907
6715 -1454
1162 -5102
-9439 -1267
9359 8724
3471 6475
-2691 -5325
-4192 4698
6101 8158
-3418 9719
