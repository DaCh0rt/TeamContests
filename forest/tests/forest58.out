77606.65 416657375.45
